generator/ServiceClientGeneratorLib/Generators/SourceFiles/StructureGenerator.tt

```diff
@@ -152,7 +152,7 @@ namespace <#=this.Config.Namespace#>.Model
                     if (member.IsExcluded)
                         continue;
 #>
-        private <#=member.DetermineType()#><#= member.IsNullable ? "?" : "" #> <#=member.VariableName#><#= member.ShouldInstantiate ? string.Format(" = new {0}();", member.DetermineType()) : ";"#>
+        private <#=member.DetermineType()#><#= member.IsNullable && !member.UseNullable ? "?" : "" #> <#=member.VariableName#><#= member.ShouldInstantiate ? string.Format(" = new {0}();", member.DetermineType()) : ";"#>
```

Comment:
The calculation for UseNullable had to be moved into DetermineType. UseNullable is the override while IsNullable is the real type of the member. This has to do with the map/list based nullable values `Dictionary<string, bool?>`  where we have to specify UseNullable in an override.

Some existing services have both which resulted in things like `int??` which was caused because the DetermineType does the UseNullable calculation now. 

The sparse test was removed after this was added to get that working but it is coming back on the next artifact update so we can only inspect the cause through offline commits until that test returns.